runtime/src/lib.rs

```diff
@@ -1277,10 +1273,6 @@ mod tests {
         // Run test
         let (executor, context) = tokio::Executor::default();
         test_blob_clone_and_concurrent_read(executor, context.clone());
-
-        // Ensure no blobs still open
-        let buffer = context.encode();
-        assert!(buffer.contains("open_blobs 0"));
```

Comment:
Any reason we don't want to assert this anymore?

Reply:
I suppose this is because of the planned change for how this should work? In the meantime, I'd be down to keep.

Reply:
I felt like it made more sense to test that metrics are updated inside `metered.rs`. I can re-add this if you want.